src/context/ContextCondition.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+
+import { withFrameworkShow } from "@/context/framework.provider";
+
+/**
+ * A simple component that displays children, but is wrapped in HOCs that add contextual conditional
+ * control. Currently only uses `withFrameworkShow`, but if this pattern is repeated in the future,
+ * further HOCs may be added.
+ */
+export const ContextCondition = withFrameworkShow(({ children }: { children: ReactNode }) => <>{children}</>);
```

Comment:
This component is very simple, but I think it will expand over time. The intent here is to provide a replacement for the If/Then and When structures from react-if when we simply want to show or hide content based on framework. There are many examples of its use in this PR.

It's primarily useful for two cases:
 * Showing or hiding multiple components in a block based on framework
 * Adding show/hide based on framework functionality to components that we didn't define ourselves (this is used heavily in the admin pages).
 
 As noted in the comment above, if we end up expanding on this pattern to provide more HOCs that control child inclusion based on some contextual information, we can add those HOCs to this component to expand its usefulness.